### ISSUE TITLE
My photos stopped showing up on my wordpress site

### ISSUE THREAD

**Issue by swesthov** [miromannino/Justified-Gallery#87]
Hello, 

Thank you for creating the plugin for displaying flickr photos on wordpress sites. For whatever reason some time again my photos quite showing up on my wordpress site (scottwesthoven.com) I"m not sure what is causing the problem, but I was hoping you could help.

Thanks,
Scott

**Comment by jrondorf** [comment]
Hi Scott, you have js errors on your WP side. Check with Google Chrome Developer Tools.

**Comment by miromannino** [comment]
I can't understand why it doesn't work just looking your site. In the page there are many errors. fancybox is loaded multiple times (not good) and despite that it is not found. JustifiedGallery is never called, ...
Consider to reduce the number of plugins and build your site just one step after the other to spot where is the plugin that destroys the others.

**Comment by swesthov** [comment]
Thanks for taking a look. I will uninstall my plugins and rebuild. 
Thanks

Sent from Outlook




On Sun, May 24, 2015 at 4:08 AM -0700, "Miro Mannino" <notifications@github.com> wrote:












I can't understand why it doesn't work just looking your site. In the page there are many errors. fancybox is loaded multiple times (not good) and despite that it is not found. JustifiedGallery is never called, ...

Consider to reduce the number of plugins and build your site just one step after the other to spot where is the plugin that destroys the others.

**Comment by swesthov** [comment]
Miro,

Thanks for your support. I uninstalled all my excessive plugins and that did the trick. I got into a bad habit of trying out different plugins, but never uninstalled them when I was done with them. Thanks once again for your help.

Scott

Scott Westhoven
scottwesthoven@gmail.com | 419-704-3284 | www.scottwesthoven.com

On May 24, 2015 at 7:08:13 AM, Miro Mannino (notifications@github.com) wrote:

I can't understand why it doesn't work just looking your site. In the page there are many errors. fancybox is loaded multiple times (not good) and despite that it is not found. JustifiedGallery is never called, ...
Consider to reduce the number of plugins and build your site just one step after the other to spot where is the plugin that destroys the others.
